playbooks/templates/.github/workflows/markdownlint.yml

```diff
@@ -27,5 +27,8 @@ jobs:
       - name: Check out code
         uses: {{ gha_checkout_action }}
 
-      - name: Run markdownlint
-        uses: actionshub/markdownlint@main
+      - name: Lint README.md
```

Comment:
Although the README.md is the most important, we will want to lint all of our .md files in the future

Reply:
markdownlint needs to specify files, it supports advanced globbing patterns like `**/*.md`, I should use it here

Reply:
As per https://github.com/igorshubovych/markdownlint-cli#globbing

Reply:
I tried running markdownlint with `**/*.md`, it finds many issues in CHANGELOG.md. And CHANGELOG.md is built from PR titles and descriptions. So we either need to run markdownlint on PR titles, or ignore CHANGELOG.md.
I would like to ignore CHANGELOG.md because 1. We do not convert it and 2. Developers see how PR desc is rendered in PRs themselves, and if the thing rendered fine in GitHub, lint issues do not matter.

Reply:
> I tried running markdownlint with `**/*.md`, it finds many issues in CHANGELOG.md. And CHANGELOG.md is built from PR titles and descriptions. So we either need to run markdownlint on PR titles, or ignore CHANGELOG.md. I would like to ignore CHANGELOG.md because 1. We do not convert it and 2. Developers see how PR desc is rendered in PRs themselves, and if the thing rendered fine in GitHub, lint issues do not matter.

I'm ok with this for now - README.html is high priority